### DISCUSSION TITLE
Website redirects to google drive

### DISCUSSION THREAD

**Discussion by L4B0MB4** [remix-run/remix#2233] [General]
When clicking on the logo on the website ( https://remix.run/docs/en/v1 ) it redirects to https://drive.google.com/drive/u/0/folders/1pbHnJqg8Y1ATs0Oi8gARH7wccJGv4I2c . I dont think that's expected is it ? 

(didn't create an issue because it's nothing wrong with remix)

**Comment by hollandThomas** [comment]
It's not a bug, it's a feature!

Most of the time when people right click an image, they want to download it. Remix ups the ante here by providing you an entire folder of assets that may suit your needs even more.

**Reply by lewsmith** [reply]
That's  cool feature. Where/how is it been implemented?

**Reply by MichaelDeBoey** [reply]
@lewsmith It's implemented just like on the RR website

https://github.com/remix-run/react-router-website/blob/1bb50210d3f9aef6251f873e05d57ba46c6a684f/app/routes/%24lang.%24ref.tsx#L147-L149
